### PR TITLE
Add support for Capability API

### PR DESCRIPTION
**Pull request by hpariwat**
Introducing support for Capability API.

Reference: https://www.omise.co/capability-api

~~Also bump the API version to 2019-05-29.~~

### CODE REVIEW

**Review by danfowler — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Review by rezigned — APPROVED** [review]

File: Omise/Client.cs [hunk 1]
```diff
@@ -7,13 +7,14 @@ namespace Omise
     public class Client
     {
         // Default to the latest API Version.
-        private string apiVersion = "2017-11-02";
```
Comment (rezigned) [inline Omise/Client.cs]: We can't change version. It'll break since fields in each version are different.

**Review by rezigned — CHANGES_REQUESTED** [review]

File: Omise/Client.cs (hunk 1 above)
Comment (hpariwat) [reply Omise/Client.cs]: The 2017 version uses a different syntax, so some significant changes have been made.

File: Omise.Examples/Examples/Capability.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public class Capability : Example
+    {
+        public async Task Retrieve__Retrieve()
+        {
+            var capability = await Client.Capability.Get();
+
+            Console.WriteLine($"supported banks: {string.Join(", ", capability.Banks)}");
+
+            foreach (var paymentMethod in capability.PaymentBackends)
+            {
```
Comment (rezigned) [inline Omise.Examples/Examples/Capability.cs:16]: We can use `KeyValuePair` for this.

```cs
foreach (KeyValuePair<string, PaymentMethod> paymentMethod in capability.PaymentBackends)
{
    var key = paymentMethod.Key;
    var value = paymentMethod.Value;
    // ...
}
```

**Review by rezigned — APPROVED** [review]

File: Omise.Examples/Examples/Capability.cs (hunk 2 above)
Comment (hpariwat) [reply Omise.Examples/Examples/Capability.cs:16]: Thanks!

**Pull request merged**